src/main/java/hw3/components/CheckBoxes.java

```diff
@@ -0,0 +1,36 @@
+package hw3.components;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+import java.util.List;
+
+public class CheckBoxes {
```

Comment:
What are these checkboxes?
I think that these checkboxes will not work for the other checkboxes except specified here

Reply:
Renamed class